main/ui/src/main/java/org/cryptomator/ui/preferences/GeneralPreferencesController.java

```diff
@@ -89,10 +90,10 @@ public void initialize() {
 
 		debugModeCheckbox.selectedProperty().bindBidirectional(settings.debugMode());
 
-		autoStartStrategy.ifPresent(autoStart -> {
-			autoStart.isAutoStartEnabled().thenAccept(enabled -> {
-				Platform.runLater(() -> autoStartCheckbox.setSelected(enabled));
-			});
+		autoStartProvider.ifPresent(autoStart -> {
+			if (autoStart.isEnabled()) {
+				Platform.runLater(() -> autoStartCheckbox.setSelected(true));
+			}
```

Comment:
Can we put this into an init method? Then we'd be in the JavaFX thread already and don't need the runLater() stuff.

Reply:
btw I'm wondering why the debugModeCheckbox part even works... The checkbox is fxml-injected and should therefore be null during construction...